cli/cmd/capture/create.go

```diff
@@ -21,13 +27,6 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/kubectl/pkg/util/i18n"
 	"k8s.io/kubectl/pkg/util/templates"
-
-	retinacmd "github.com/microsoft/retina/cli/cmd"
-	retinav1alpha1 "github.com/microsoft/retina/crd/api/v1alpha1"
-	pkgcapture "github.com/microsoft/retina/pkg/capture"
-	captureConstants "github.com/microsoft/retina/pkg/capture/constants"
-	captureUtils "github.com/microsoft/retina/pkg/capture/utils"
-	"github.com/microsoft/retina/pkg/config"
```

Comment:
I actually kinda like this separation, since it has 1) stdlib 2) external packages 3) intra-module packages.

Reply:
reasonable, but we need the formatter to do it